core/labels.go

```diff
@@ -36,10 +36,15 @@ var (
 		Key:         "pod_name",
 		Description: "The name of the pod",
 	}
+	// TODO: Replace with namespace_name
```

Comment:
Fixed. Although I'm not sure what consequences this change will have for GKE/GCM. We will have to put this in release notes.